gm4_garden_variety/pack.mcmeta

```diff
@@ -0,0 +1,36 @@
+{
+    "pack": {
+        "pack_format": 8,
+        "description": [
+            "Garden Variety",
+            "\n",
+            {
+                "text": "Gamemode 4 for 1.18",
+                "color": "#4AA0C7"
+            }
+        ]
+    },
+    "module_name": "Garden Variety",
+    "module_id": "garden_variety",
+    "site_description": "Adds a variety of new trees with unique generation.",
+    "site_categories": [],
+    "video_link": "",
+    "wiki_link": "https://wiki.gm4.co/wiki/Garden_Variety",
+    "required_modules": [
+        "custom_crafters",
+        "lib_lore"
+    ],
+    "recommended_modules": [
+        "orbis",
+        "podzol_rooting_soil",
+        "fruiting_trees"
```

Comment:
Fruiting trees is not a module, rather it should say it recommends apple trees. Why is apply trees not a required module? Does it implement its own custom tree and sapling logic? If so, that seems like a waste and makes things harder to maintain. Making fruiting trees a library may make sense.

Reply:
garden_variety does not require fruiting_tree mechanics however if it is enabled it does add additional functionality in the form of some trees having the chance of spawning with fruit leaves.

When I first made the module, fruiting_trees was required. However, for a variety of reasons I decided to remove this requirement.
- I have my own system for dropping saplings from the leaves that is more efficient. fruiting_trees uses armor stands for each leaf that has fruit, however, I use a single leaf for each tree that uses a marker.
- I don't remember the specifics so maybe I am misremembering. I remember running into problems when it came to merging the mutation system into sapling placement and breaking. Because of this I decided to just make my own system for handling saplings.
- I would love it if fruiting_trees was its own library because I do not like the idea of forcing the player to use apple_trees because fruiting_trees funcationality is required.